### DISCUSSION TITLE
learn-shell frontend Loop

### DISCUSSION THREAD

**Discussion by palsachennaiah** [learndevopsonline/learndevopsonline#1629] [Q&A]
![Screenshot 2023-11-22 182808](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/1c6a28dc-7c0b-4675-9ae5-b108b3488306)
Here Fruit names I'm not getting

**Comment by palsachennaiah** [comment]
![Screenshot 2023-11-22 184629](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/bb85fc12-d908-4df0-baab-daa2c59453d7)
Ok i got above one

**Reply by sreedharm07** [reply]
ok
